seirmo/apps/simulation_app.py

```diff
@@ -0,0 +1,76 @@
+#
+# This file is part of seirmo (https://github.com/SABS-R3-Epidemiology/seirmo/)
+# which is released under the BSD 3-clause license. See accompanying LICENSE.md
+# for copyright notice and full license details.
+#
+
+import pandas as pd
+import seirmo as se
+
+import dash
+import dash_bootstrap_components as dbc
+import dash_core_components as dcc
+
+
+class SimulationApp(object):
+    """SimulationApp Class:
+
+    Creates the SEIR model simulation app.
+
+    """
+
+    def __init__(self):
+        super(SimulationApp, self).__init__()
+
+        self.IncidenceNumberPlot = se.IncidenceNumberPlot()
+        self.SliderComponent = se._SliderComponent()
+
+        self.app = dash.Dash(external_stylesheets=[dbc.themes.BOOTSTRAP])
+
+        self.app.layout = dbc.Container(
+            dbc.Row(
+                [
+                    dbc.Col(
+                        dcc.Graph(
+                            figure=self.IncidenceNumberPlot._fig, id='fig')),
+                    dbc.Col(
+                        [dbc.Row(
+                            self.SliderComponent.group_sliders(
+                                self.slider_ids()[:4], 'init_value')),
+                         dbc.Row(
+                            self.SliderComponent.group_sliders(
+                                self.slider_ids()[4:], 'constant'))]
+                    )
+                ]
+            )
+        )
+
+        self.simulation_start = 0
+        self.simulation_end = 50
+
+    def add_data(self, data):
```

Comment:
similar to the add_data method of the figure class, allow to specify the x and y labels and pass those to the figure class arguents